src/chanjo2/endpoints/samples.py

```diff
@@ -1,45 +1,81 @@
 from pathlib import Path
 from typing import List
 
+from chanjo2.constants import BAD_REQUEST, NOT_FOUND
+from chanjo2.crud import samples as crud_samples
 from chanjo2.dbutil import get_session
-from chanjo2.meta.handle_bed import parse_bed
-from chanjo2.models.pydantic_models import CoverageInterval, SampleCreate, SampleRead
-from fastapi import APIRouter, Depends, File, HTTPException, Query, status
-from sqlmodel import Session, select
+from chanjo2.models import pydantic_models, sql_models
+from fastapi import APIRouter, Depends, HTTPException, status
+from sqlmodel import Session
 
 router = APIRouter()
 
 
-@router.post("/sample_create/", response_model=SampleRead)
-def create_sample(*, session: Session = Depends(get_session), sample: SampleCreate):
+### Case endpoints
+@router.post("/cases/", response_model=pydantic_models.CaseRead)
```

Comment:
```suggestion
@router.post("/case/", response_model=CaseRead)
```
Since it only seems to a single case

Reply:
As as the other comment. I wanted to follow the conventions